app.py

```diff
@@ -64,23 +62,89 @@ def download_chapter(url):
     """Download a chapter."""
     return requests.get(url).content
 
+def yes_or_no(question):
```

Comment:
[Click prompt](http://click.palletsprojects.com/en/7.x/prompts/) is a better alternative here, no need for custom function.

Reply:
Yes, same as above